java/simulator/src/main/java/com/hazelcast/simulator/utils/SimulatorHistogramLogProcessor.java

```diff
@@ -18,12 +18,13 @@
 import org.HdrHistogram.DoubleHistogram;
 import org.HdrHistogram.Histogram;
 
-import java.io.FileNotFoundException;
+import java.io.Closeable;
+import java.io.IOException;
 
 @SuppressWarnings({"checkstyle:methodlength", "checkstyle:magicnumber"})
-public class SimulatorHistogramLogProcessor extends HistogramLogProcessor {
+public class SimulatorHistogramLogProcessor extends HistogramLogProcessor implements Closeable {
 
-    public SimulatorHistogramLogProcessor(String[] args) throws FileNotFoundException {
```

Comment:
why do you think this is better exception?

Reply:
`FileNotFoundException` is derived from `IOException`. If a method throws IOException, it could also throw a FileNotFoundException as well as other exceptions which are subtype of IOException. So probably having broader scope here would be better.